wandb/cli/cli.py

```diff
@@ -1144,7 +1214,13 @@ def launch(
 
 @cli.command(context_settings=CONTEXT, help="Run a W&B launch agent (Experimental)")
 @click.pass_context
-@click.argument("project", nargs=1, required=False)
+@click.option(
+    "--project",
+    "-p",
+    default=None,
+    help="Name of the target project which the new run will be sent to. "
+    "If passed in, will override the project value passed in using a config file.",
```

Comment:
this is the project that the agent will watch rather than the project the run will be sent to. (They are one in the same right now but won't alwasy be)